src/index.ts

```diff
@@ -274,7 +274,7 @@ export class MidiOut {
  * Unique identifier for a note message in a specific channel.
  */
 function noteIdentifier(event: NoteMessageEvent) {
-  return event.note.number + 128 * event.message.channel;
+  return event.note.number + 128 * (event.message.channel - 1); // webmidi sends channels 1-16, but identifier only needs to range between 0 and (16 * 128) - 1 = 2047
```

Comment:
This is just a unique identifier. It doesn't matter how it looks. Please don't write code that depends on it.